broker/broker-route-manager/src/main/java/org/codice/ddf/broker/route/manager/DynamicRouteDeployer.java

```diff
@@ -32,15 +32,26 @@
 public class DynamicRouteDeployer implements ArtifactInstaller {
   private static final Logger LOGGER = LoggerFactory.getLogger(DynamicRouteDeployer.class);
 
-  private final CamelContext camelContext;
+  /**
+   * This map is made static in order to keep a reference of the map entries after a restart of this
+   * bundle. This is because a new instance of the DynamicRouteDeployer is created when the bundle
+   * gets restarted and the reference to the existing map entries would be cleaned up, therefore
+   * this map should be made static.
+   */
+  private static Map<File, List<RouteDefinition>> processedFiles = new ConcurrentHashMap<>();
 
-  private Map<File, List<RouteDefinition>> processedFiles = new HashMap<>();
+  private final CamelContext camelContext;
 
   public DynamicRouteDeployer(CamelContext camelContext) {
     this.camelContext = camelContext;
   }
 
-  public void init() {}
+  public void init() throws Exception {
+    processedFiles
+        .values()
+        .stream()
```

Comment:
You could call `forEach(...)` on the collection returned by `values()` without having to `stream()` first. Also, you could use a method reference for `addRouteDefinitions`.

Reply:
Fixed.